app/sdk/utils/Constants.java

```diff
@@ -34,4 +35,6 @@
     public static final String UPDATED_DATE_PARAM = "updatedate";
 
     public static final String AllowedOffsetKey = "apptree.allowedServerOffset";
+
+    public static final String BRANDING_ROUTER_URL = "https://go-monolith-v2-sbx.herokuapp.com";
```

Comment:
Rename this to `ROUTER_URL`.

Shouldn't the default be `https://go-monolith-v2.herokuapp.com`?

Reply:
addressed